admin/deployments.go

```diff
@@ -170,6 +170,46 @@ func (s *Service) updateDeployment(ctx context.Context, depl *database.Deploymen
 	return nil
 }
 
+// HIBERNATE free deployments
+func (s *Service) HibernateDeployments(ctx context.Context) error {
+	// Not checking any permissions as its use only internally by scheduled job, will add if required
+	projects, err := s.DB.FindAllProjects(ctx)
+	if err != nil {
+		return err
+	}
+
+	for _, proj := range projects {
+		depl, err := s.DB.FindDeployment(ctx, *proj.ProdDeploymentID)
+		if err != nil {
+			fmt.Printf("depl not found for proj %s", proj.Name)
+			continue
+		}
```

Comment:
Instead of `FindAllProjects`, add a DB call that returns all deployments where `used_on + prod_ttl < now()`